plugins/meta/portmap/README.md

```diff
@@ -8,6 +8,8 @@ You should use this plugin as part of a network configuration list. It accepts
 the following configuration options:
 
 * `snat` - boolean, default true. If true or omitted, set up the SNAT chains
+* `markMasqBit` - int, (0-31), default 13. The mark bit to use for masquerading (see section SNAT)
+* `externalSetMarkChain` - string, default nil. If you already have a Masquerade mark chain (e.g. Kubernetes), specify it here. This will use that instead of creating a separate chain. This cannot be used with `markMasqBit`
```

Comment:
I think the last sentence would be better if it said something like "If this is specified, markMasqBit is ignored".  Cannot be used isn't super clear - will the server catch fire?

Reply:
Right now the plugin errors out if both are set; thoughts?

Reply:
I'm a little uneasy about that - if someone wants to set the chain, they have to go and look that the bit is pointed and they can nil it - nothing about this description suggests it can also be nilled.  Maybe just an extra line that tells people to nil / omit it?